README.md

```diff
@@ -1,3 +1,295 @@
+
+
+
+### landscape.etcd
+```yaml
+etcd:
+  backup:
+    type: <gcs|s3|abs>                  # type of blob storage
+    resourceGroup: ...                  # Azure resource group you would like to use for your backup
+    region: (( iaas.region ))           # region of blob storage (default: same as above)
+    credentials: (( iaas.credentials )) # credentials for the blob storage's IaaS provider (default: same as above)
+```
+Configuration of what blob storage to use for the etcd key-value store. If your IaaS provider offers a blob storage you can use the same values for `etc.backup.region` and `etc.backup.credentials` as above for `iaas.region` and `iaas.credentials` correspondingly by using the [(( foo ))](https://github.com/mandelsoft/spiff/blob/master/README.md#-foo-) expression of spiff.
+
+| Field | Type | Description | Example&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; | Iaas Provider Documentation |
+|:------|:--------|:--------|:--------|:---------|
+|`backup.type`|Fixed value| Type of your blob store. Supported blob stores: `gcs` ([Google Cloud Storage](https://cloud.google.com/storage/)), `s3` ([Amazon S3](https://aws.amazon.com/s3/)), and `abs`[Azure Blob Storage](https://docs.microsoft.com/en-us/azure/storage/blobs/storage-blobs-overview).|`gcs`|n.a.|
+|`backup.resourceGroup`|IaaS provider specific |Azure specific. Create an Azure blob store first which uses a resource group. Provide the resource group here. | `my-Azure-RG` | [Azure](https://docs.microsoft.com/en-us/azure/storage/common/storage-quickstart-create-account?tabs=azure-portal) (HowTo) |
+|`backup.region`|IaaS provider specific|Region of blob storage. |`(( iaas.region ))` |[GCP (overview)](https://cloud.google.com/docs/geography-and-regions), [AWS (overview)](https://docs.aws.amazon.com/general/latest/gr/rande.html)|
+|`backup.credentials`|IaaS provider specific|Service account credentials in a provider-specific format. |`(( iaas.creds ))` |[GCP](https://cloud.google.com/iam/docs/creating-managing-service-account-keys#creating_service_account_keys), [AWS](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_users.html#id_users_service_accounts), [Azure](https://docs.microsoft.com/en-us/rest/api/storageservices/authorization-for-the-azure-storage-services)|
+
+
+### landscape.dns
+```yaml
+dns:
+  type: <google-clouddns|aws-route53>       # dns provider
+  hostedZoneID:                             # hosted zone id for chosen domain
+  credentials:                              # credentials for the dns provider
+```
+Configuration for the Domain Name Service (DNS) provider. If your IaaS provider offers a DNS you can use the same values for `dns.credentials` as for `iaas.creds` above by using the [(( foo ))](https://github.com/mandelsoft/spiff/blob/master/README.md#-foo-) expression of spiff.
```

Comment:
If the hosted zone for your selected dns base name (in cluster) is owned by your IAAS account, the same credentials can be used. If the belong to another account (or another iass provider) the appropriate credentials (and type) have to be configured.

Reply:
ok, thanks, I made this more explicit now.